src/mixed.js

```diff
@@ -341,13 +346,15 @@ SchemaType.prototype = {
    */
   test(...args) {
     let opts = args[0];
+
+    // TODO: search all method calls and use hash as the argument
     if (args.length > 1) {
-      let [name, message, test] = args;
+      let [name, message, test, localePath] = args;
```

Comment:
i wouldn't support this option in anything other than the options object call signature, this is mostly an internal option anyway